index.html

```diff
@@ -24,14 +24,205 @@ <h1 class="headline1">Hey There.<br>I'm Femi</h1>
                 don’t hestiate to contact me.</p>
             </div>
             <ul class="container">
-                <li><img src="Images/github.png" alt="github"></li>
-                <li><img src="Images/Linkedinicon.png" alt="linkedin"></li>
-                <li><img src="Images/angellist.png" alt="Angel list"></li>
-                <li><img src="Images/twitter.png" alt="Twitter"></li>
-                <li><img src="Images/medium.png" alt="Medium"></li>
+                <li><a href="#"><img src="Images/github.png" alt="github"></a></li>
+                <li><a href="#"><img src="Images/Linkedinicon.png" alt="linkedin"></a></li>
+                <li><a href="#"><img src="Images/angellist.png" alt="Angel list"></a></li>
+                <li><a href="#"><img src="Images/twitter.png" alt="Twitter"></a></li>
+                <li><a href="#"><img src="Images/medium.png" alt="Medium"></a></li>
             </ul>
             <img class="backgroundImage" src="Images/Illustration-header@2x.png" alt="background image">
             <img class="toggleImage" src="Images/Group.png" alt="background image">
         </section>
+        <section class="work">
+            <h3 class="workhead">My Recent Works</h3>
+            <p class="workhead2">My Recent Works</p>
```

Comment:
- [ ] The title (My Recent Works) should use a header tag(e.g. `h2`, `h3`, `h4`) as defined in the design